postman_tests/lib/es.py

```diff
@@ -0,0 +1,97 @@
+from __future__ import absolute_import
```

Comment:
Instead of `es.py` is should be called `ds.py` or `datastore.py` because we never interact with `ES` directly